src/uris/deepLinks/deepLink.ts

```diff
@@ -387,6 +391,7 @@ export const deepLinkStateTransitionTable: Record<string, Record<string, DeepLin
 		[DeepLinkServiceAction.OpenGraph]: DeepLinkServiceState.OpenGraph,
 		[DeepLinkServiceAction.OpenFile]: DeepLinkServiceState.OpenFile,
 		[DeepLinkServiceAction.OpenSwitch]: DeepLinkServiceState.SwitchToRef,
+		[DeepLinkServiceAction.OpenAllPrChanges]: DeepLinkServiceState.OpenAllPrChanges,
```

Comment:
This transition is no longer necessary since it would/should only be reachable from `OpenInspect`